### ISSUE TITLE
npx init fails on Windows

### ISSUE THREAD

**Issue by Tyler-RCSD** [keystonejs/create-keystone-next-app#39]
Hello,

I'm trying to follow the [getting started](https://next.keystonejs.com/tutorials/getting-started-with-create-keystone-next-app) guide but am running into a wall.  When I run `npx create-keystone-next-app` it throws an error (no such file or directory) after filling in the requested info.  

Windows 10
Node 10.16.3
npm 6.9.0

Screenshot:
![image](https://user-images.githubusercontent.com/42674425/112046672-f3726f00-8b11-11eb-9877-dac431642d85.png)

I believe I meet all the node/npm version requirements as laid out.  Not sure what's going wrong.

**Comment by fodurrr** [comment]
@Tyler-RCSD same here, Windows 10, Node v14.15.4, NPM 6.14.11.
I tried and yard does not work either.

May be .gitignore file is really missing from CLI? BTW I checked the `C:\Users\Peti\AppData\Roaming\npm-cache\_npx\19476\node_modules\create-keystone-next-app\starter` folder and it is totally empty.

**Comment by fodurrr** [comment]
@jesstelford @mitchellhamilton Hi, can somebody check or help us to try out keystone? CLI is not working on Windows, see above.

**Comment by jesstelford** [comment]
Hey @fodurrr Thanks for confirming this issue. I've passed this on to the folks who are woking on this part of the codebase. Hopefully they can debug and find a solution for you.

In the meantime, can you try remove the directory `C:\Users\Peti\AppData\Roaming\npm-cache\_npx\19476\node_modules\create-keystone-next-app\starter`, then run the command again?

If that still doesn't work, you can download just the [`starter` directory](https://github.com/keystonejs/create-keystone-next-app/tree/main/create-keystone-next-app/starter) directly using this link: https://downgit.github.io/#/home?url=https://github.com/keystonejs/create-keystone-next-app/tree/main/create-keystone-next-app/starter, then:
1. Unzip that folder (iut's the same folder you would otherwise get by running `npx create-keystone-next-app`)
2. Edit `keystone.ts` to replace lines 18 & 37 with actual values
3. Run `npm install` to install the dependencies
4. Run `npm run dev`

**Comment by fodurrr** [comment]
> Hey @fodurrr Thanks for confirming this issue. I've passed this on to the folks who are woking on this part of the codebase. Hopefully they can debug and find a solution for you.
> 
> In the meantime, can you try remove the directory `C:\Users\Peti\AppData\Roaming\npm-cache\_npx\19476\node_modules\create-keystone-next-app\starter`, then run the command again?
> 
> If that still doesn't work, you can download just the [`starter` directory](https://github.com/keystonejs/create-keystone-next-app/tree/main/create-keystone-next-app/starter) directly using this link: https://downgit.github.io/#/home?url=https://github.com/keystonejs/create-keystone-next-app/tree/main/create-keystone-next-app/starter, then:
> 
> 1. Unzip that folder (iut's the same folder you would otherwise get by running `npx create-keystone-next-app`)
> 2. Edit `keystone.ts` to replace lines 18 & 37 with actual values
> 3. Run `npm install` to install the dependencies
> 4. Run `npm run dev`

I tried and deleted the roaming folder before but it did not help. It creates a new `..._npx\{random number}\...` folder anytime you start the CLI  but the starter folder is empty.

Thanks for the manual download tip :)

**Comment by dominikwilkowski** [comment]
Hi @Tyler-RCSD and @fodurrr 
I had a look into this and am not able to reproduce this.
I'm very curious especially because @fodurrr mentioned that the starter folder was empty in `C:\Users\Peti\AppData\Roaming\npm-cache\_npx\19476\node_modules\create-keystone-next-app\starter`.
Can either of you help me out and try this to see if this is an npm caching issue:

Please install `create-keystone-next-app` globally and run it then via the command line.
```sh
npm install create-keystone-next-app -g
```
After the install is finished and in the folder you want to create your project in:
```sh
create-keystone-next-app
```
Let me know if that works at all. Thank you.

**Comment by Tyler-RCSD** [comment]
Hi @dominikwilkowski.  I installed it globally as you suggested and was met with the same result:

![image](https://user-images.githubusercontent.com/42674425/112479336-91e11900-8d3a-11eb-9874-90c7f27a9b68.png)

The folder mentioned in that screenshot is `C:\Users\me\AppData\Roaming\npm\node_modules\create-keystone-next-app\starter` this time and if I go there this is the listing:

![image](https://user-images.githubusercontent.com/42674425/112479599-d371c400-8d3a-11eb-8c0c-136e627113b7.png)

Looks like it's trying to copy .gitignore and that file not in the folder.

**Comment by dominikwilkowski** [comment]
Thank you @Tyler-RCSD that helps a lot.
Let me get back to you when we have a fix.

**Comment by dominikwilkowski** [comment]
Ok @Tyler-RCSD and @fodurrr 

I found a fix that I have tested in my VM.
this change has been merged now and will be published on Monday.
Let me check in with you then again to see if this all worked as I hope.

Thank you for reporting this.

**Comment by dominikwilkowski** [comment]
Ok the package has been published now to version 1.2.0.
Please let me know if you find any issues with the solution.
